src/index.js

```diff
@@ -228,41 +228,58 @@ class App extends React.Component {
           onMouseDown={e => {
             const x = e.clientX - e.target.offsetLeft;
             const y = e.clientY - e.target.offsetTop;
-            const element = newElement(this.state.elementType, x, y);
 
```

Comment:
This is really interesting approach to put it onMouseDown. In my mind I was thinking that we would have to do onMouseUp and detect whether we had moved and trigger the click behavior. I can’t see anything wrong on top of my head where it wouldn’t work so let’s ship it :)